### PR TITLE
Feat/ethereumjs upgrade

### PR DESCRIPTION
**Pull request by nimraarif21**

### CODE REVIEW

File: modules/account-lib/src/coin/eth/types.ts [hunk 1]
```diff
@@ -76,9 +76,9 @@ export class EthTransactionData implements EthLikeTransactionData {
 
     if (this.tx.verifySignature()) {
       result.from = bufferToHex(this.tx.getSenderAddress().toBuffer());
-      result.v = bufferToHex(this.tx.v);
-      result.r = bufferToHex(this.tx.r);
-      result.s = bufferToHex(this.tx.s);
+      result.v = bufferToHex(toBuffer(this.tx.v))
```
Comment (CapnMigraine) [inline modules/account-lib/src/coin/eth/types.ts:79]: missing semi-colon
```suggestion
      result.v = bufferToHex(toBuffer(this.tx.v));
```

File: modules/account-lib/src/coin/eth/utils.ts [hunk 2]
```diff
@@ -260,12 +260,12 @@ export function decodeTokenTransferData(data: string): TokenTransferData {
   );
 
   return {
-    to: addHexPrefix(to),
+    to: addHexPrefix(to.toString('hex')),
```
Comment (CapnMigraine) [inline modules/account-lib/src/coin/eth/utils.ts:263]: is this not already a hex string?
same question for other conversions below

**Review by CapnMigraine — COMMENTED** [review]
Please run yarn install with node v.14.16.0 and add the changes made to `yarn.lock`

LGTM overall, I'll let the @BitGo/eth-team give the final approval.

File: modules/account-lib/src/coin/eth/utils.ts (hunk 2 above)
Comment (nimraarif21) [reply modules/account-lib/src/coin/eth/utils.ts:263]: No this is a buffer. We need to convert it to string and its true for all conversions below.

File: modules/account-lib/src/coin/eth/types.ts (hunk 1 above)
Comment (nimraarif21) [reply modules/account-lib/src/coin/eth/types.ts]: Its updated now

**Comment by 0xabrarhussain** [comment]
closing stale PR

**Pull request closed without merging**